### PR TITLE
Feat html t1003

### PR DESCRIPTION
**Pull request by sganeshkumaar**

### CODE REVIEW

**Review by lakshmipooja1510 — CHANGES_REQUESTED** [review]

File: src/bottom.css [hunk 1]
```diff
@@ -1,3 +1,158 @@
 .bottom-section{
     color: white;
+    margin: 80px auto;
+    width: 90vw;
+}
+
+.bottom-head{
+    align-items: flex-end;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:8]: align flex-end property removed

File: src/bottom.css [hunk 2]
```diff
@@ -1,3 +1,158 @@
...
+    display:flex;
+    height:fit-content;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:10]: removed the height property

File: src/bottom.css [hunk 3]
```diff
@@ -1,3 +1,158 @@
...
+    justify-content: space-between;
+    margin: auto;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:12]: i used margin for centering the div but now ive changed it to flex properties

File: src/bottom.css [hunk 4]
```diff
@@ -1,3 +1,158 @@
...
+    width: fit-content;
+}
+
+ 
+
+.tile-factors{
+    align-items: flex-end;
+    display: flex;
+    flex-direction: column;
+    height: 95%;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:81]: value changed to px

File: src/bottom.css [hunk 5]
```diff
@@ -1,3 +1,158 @@
...
+    justify-content: space-between;
+    width: fit-content;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:83]: width removed in tile-city-details and tile-factors

File: src/bottom.css [hunk 6]
```diff
@@ -1,3 +1,158 @@
...
+}
+
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:85]: empty lines removed in bottom.css

File: src/bottom.css [hunk 7]
```diff
@@ -1,3 +1,158 @@
...
+ 
+
+.tile-continent{
+    color:#FFF461;
+    font-size: 20px;
+    font-weight: 400;
+}
+
+.tile-city-time{
+    color: #FFF;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:95]: text color property removed

File: src/bottom.css [hunk 8]
```diff
@@ -1,3 +1,158 @@
...
+    align-items:center;
+    display: flex;
+    font-size: 20px;
+    font-weight:400;
+}
+
+.tile-humidity-img{
+    height:21px;
+    margin-right: 4px;
+    width: auto;
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/bottom.css:115]: width property removed

File: src/weather.html [hunk 9]
```diff
@@ -365,6 +365,216 @@
...
+                            <div class="tile-temp">32 &degC</div>
+                            <div class="tile-humidity">
+                                <img src="../Assets/HTM&CSS/Weather_Icons/humidityIcon.svg" alt=""class="tile-humidity-img">
+                                <span class="tile-humitdity-text">53%</span>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+
```
[the remark this answers is not in the document]
Comment (sganeshkumaar) [reply src/weather.html:577]: empty line removed

File: src/bottom.css [hunk 10]
```diff
@@ -1,3 +1,165 @@
+/* styling for entire bottom secion */
 .bottom-section{
     color: white;
+    display:flex;
```
Comment (lakshmipooja1510) [inline src/bottom.css:4]: do we need flex and flex-direction

File: src/bottom.css [hunk 11]
```diff
@@ -1,3 +1,165 @@
...
+    flex-direction: column;
+    margin: 80px auto;
+    width: 90vw;
+}
+
+/* styling for heading section for bottom */
+.bottom-head{
+    align-self: center;
```
Comment (lakshmipooja1510) [inline src/bottom.css:12]: Do we need align-self and width property

**Review by lakshmipooja1510 — CHANGES_REQUESTED** [review]

File: src/bottom.css (hunk 10 above)
Comment (sganeshkumaar) [reply src/bottom.css:4]: i gave the bottom section as flex to center the bottom head

File: src/bottom.css (hunk 11 above)
Comment (sganeshkumaar) [reply src/bottom.css:12]: this centers, only the bottom head on center
image

Comment (sganeshkumaar) [reply src/bottom.css:11]: yes we also need width property to make the head text and icons go apart

**Review by lakshmipooja1510 — APPROVED** [review]

**Pull request merged**